.vscode/settings.json

```diff
@@ -8,8 +8,8 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.organizeImports": true
+    "source.fixAll.eslint": "explicit",
```

Comment:
Where these intentional? @neilcampbell

Reply:
Yeah the boolean option has been deprecated for a while, and VS Code automatically makes this change now.